### PR TITLE
footer fixed 

### LINKED ISSUE #11: Improvement in the footer section
*the footer looks very blank and and is less interactive , I would like to create a better UI with enhanced features . 

## Please assign this issue to me so that I can start working over it

### PR DESCRIPTION
**Pull request by kairveeehh**
#11  fixed the footer added all the social icons and copyright 
## please review and merge @Its-Aman-Yadav 
Screenshot 2024-05-12 170402

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -231,8 +231,66 @@ <h2>Open Source Village</h2>
...
+            </h2>
+          </div>
+          <p class="abc" style="color: whitesmoke;">
+Join the Community now and become a Villager.
+          </p>
+        </div>
+        <div class="footer_box">
+          <h4 class="footer_title" style="color: whitesmoke;">Company</h4>
+          <ul class="footer_links">
+            <a href="/#">
```
Comment (Its-Aman-Yadav) [inline index.html:254]: Could you please remove these links which have no work? or refer to #

**Review by Its-Aman-Yadav — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (kairveeehh) [reply index.html:254]: Sure also if you want I can link these to the profile pages of the project if any

Comment (kairveeehh) [reply index.html:245]: @Its-Aman-Yadav
##   please review and merge it now

**Comment by kairveeehh** [comment]
@Its-Aman-Yadav   please review and merge this

**Comment by Its-Aman-Yadav** [comment]
hey @kairveeehh please remove the links that are not working
Thanks

**Comment by kairveeehh** [comment]
> hey @kairveeehh please remove the links that are not working Thanks

@Its-Aman-Yadav I did it will recheck and create a new PR to ensure it goes fine this time

**Pull request closed without merging**